lib/src/pie/PieSliceAngularInset.tsx

```diff
@@ -2,30 +2,40 @@ import React from "react";
 import { type Color, Path, type PathProps } from "@shopify/react-native-skia";
 import { useSliceAngularInsetPath } from "./hooks/useSliceAngularInsetPath";
 import { usePieSliceContext } from "./contexts/PieSliceContext";
+import type { PathAnimationConfig } from "../hooks/useAnimatedPath";
+import { AnimatedPath } from "../cartesian/components/AnimatedPath";
 
 export type PieSliceAngularInsetData = {
   angularStrokeWidth: number;
   angularStrokeColor: Color;
 };
 
-type AdditionalPathProps = Partial<Omit<PathProps, "color" | "path">>;
+type AdditionalPathProps = Partial<Omit<PathProps, "color" | "path">> & {
+  animate?: PathAnimationConfig;
+};
 
 type PieSliceAngularInsetProps = {
   angularInset: PieSliceAngularInsetData;
 } & AdditionalPathProps;
 
 export const PieSliceAngularInset = (props: PieSliceAngularInsetProps) => {
-  const { angularInset, children, ...rest } = props;
+  const { angularInset, children, animate, ...rest } = props;
   const { slice } = usePieSliceContext();
   const [path, insetPaint] = useSliceAngularInsetPath({ slice, angularInset });
 
+  // If the path is empty, don't render anything
+  if (path.toSVGString() === "M0 0L0 0M0 0L0 0") {
+    return null;
+  }
+
   if (angularInset.angularStrokeWidth === 0) {
     return null;
   }
 
+  const Component = animate ? AnimatedPath : Path;
   return (
-    <Path path={path} paint={insetPaint} {...rest}>
+    <Component path={path} paint={insetPaint} animate={animate} {...rest}>
```

Comment:
thinking out loud. Would be nice in future cleanup if we could just always use `AnimatedPath` so we don't have to always check which component to render.

Reply:
Agreed. There's potentially a perf hit of some sort, but it would definitely keep things a bit simpler.